### PR TITLE
Manual tweaks to fix resulting autoformatting.

### PR DESCRIPTION
**Pull request by houglum**

### CODE REVIEW

File: gslib/commands/cp.py [hunk 1]
```diff
@@ -974,20 +984,20 @@ def CopyFunc(self, copy_object_info, thread_state=None, preserve_posix=False):
...
+          gsutil_api,
+          self,
+          _CopyExceptionHandler,
+          src_obj_metadata=src_obj_metadata,
+          allow_splitting=True,
+          headers=self.headers,
+          manifest=self.manifest,
+          gzip_encoded=self.gzip_encoded,
+          gzip_exts=self.gzip_exts,
+          preserve_posix=preserve_posix)
```
Comment (catleeball) [inline gslib/commands/cp.py:1000]: Were you intending a trailing comma here too?

File: gslib/commands/cp.py [hunk 2]
```diff
@@ -1075,9 +1085,11 @@ def _ConstructNameExpansionIteratorDstTupleIterator(self, src_url_strs_iter,
...
       # that the destination directory does not exist and copy the source
       # directories to the destination directory.
-      (exp_dst_url,
-       have_existing_dst_container) = (copy_helper.ExpandUrlToSingleBlr(
-           dst_url_str, self.gsutil_api, self.project_id, logger=self.logger))
+      exp_dst_url, have_existing_dst_container = (
+          copy_helper.ExpandUrlToSingleBlr(dst_url_str,
+                                           self.gsutil_api,
+                                           self.project_id,
+                                           logger=self.logger))
```
Comment (catleeball) [inline gslib/commands/cp.py:1092]: Likewise here, should this have a final trailing comma?

File: gslib/utils/boto_util.py [hunk 3]
```diff
@@ -534,12 +540,16 @@ def _PatchedShouldRetryMethod(self, response, chunked_transfer=False):
     # If ``RequestTimeout`` is present, we'll retry. Otherwise, bomb
     # out.
     body = response.read()
-    err = provider.storage_response_error(response.status, response.reason,
-                                          body)
+    err = provider.storage_response_error(
+        response.status,
+        response.reason,
+        body
```
Comment (catleeball) [inline gslib/utils/boto_util.py:546]: Trailing comma intended here?

**Review by catleeball — COMMENTED** [review]
Left some double-checking nits from space-split lists that didn't have terminating commas.

Also, did you run `yapf` after your changes? Just to be sure Travis doesn't complain later, or that `yapf` doesn't clobber your nice formatting later. :)

**Comment by houglum** [comment]
Yep, I made sure to run it before committing :)

Addressed comments, PTAL.

**Comment by catleeball** [comment]
LGTM! Merged, tyvm!

**Pull request merged**